pkg/ccl/changefeedccl/sink_kafka.go

```diff
@@ -134,6 +134,37 @@ type kafkaSink struct {
 	disableInternalRetry bool
 }
 
+var saramaCompressionCodecOptions = map[string]sarama.CompressionCodec{
+	"none":   sarama.CompressionNone,
+	"gzip":   sarama.CompressionGZIP,
+	"snappy": sarama.CompressionSnappy,
+	"lz4":    sarama.CompressionLZ4,
+	"zstd":   sarama.CompressionZSTD,
+}
+
+func validateCompressionCodec(s string) (sarama.CompressionCodec, error) {
+	codec, ok := saramaCompressionCodecOptions[s]
```

Comment:
do we need to lower case?

Reply:
I'll make them all upper. We use that for "RequiredAcks"